src/app/InteractionModelEngine.h

```diff
@@ -164,6 +164,17 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
 
 void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId,
                                   chip::TLV::TLVReader & aReader, Command * apCommandObj);
+
+/**
+ *  Check whether the given cluster exists on the given endpoint and supports the given command.
+ *  TODO: The implementation lives in ember-compatibility-functions.cpp, this should be replaced by IM command catalog look up
+ * function after we have a cluster catalog in interaction model engine.
+ *  TODO: The endpoint id on response command (client side command) is unclear, so we don't have a ClientClusterCommandExists
+ * function. (Spec#3258)
+ *
+ *  @retval  If the endpoint contains the cluster and command.
```

Comment:
```suggestion
 *  @retval  True if the endpoint contains the server side of the given cluster and that cluster implements the given command, false otherwise.
```